firestore-bigquery-export/firestore-bigquery-change-tracker/src/bigquery/index.ts

```diff
@@ -137,7 +196,11 @@ export class FirestoreBigQueryEventHistoryTracker
     const expectedErrors = [
       {
         message: "no such field.",
-        location: "document_id",
```

Comment:
Is there a scenario where `document_id ` could still be a potential error field? Addition rather than removing?

Reply:
Nothing was removed. Just used the variable name below to be more type safe.